src/app/3d/qgs3dmapcanvaswidget.cpp

```diff
@@ -463,13 +471,30 @@ void Qgs3DMapCanvasWidget::measureLine()
   mCanvas->setMapTool( action->isChecked() ? mMapToolMeasureLine : nullptr );
 }
 
-void Qgs3DMapCanvasWidget::changePointCloudAttribute()
+void Qgs3DMapCanvasWidget::paintBrush()
```

Comment:
How about this? it is a bit long, but this is a class not specific to point clouds...
```suggestion
void Qgs3DMapCanvasWidget::changePointCloudAttributeByPaintbrush()
```